src/codecs/framers/character_delimited.rs

```diff
@@ -53,7 +53,10 @@ impl CharacterDelimitedCodec {
         }
     }
 
-    /// Returns a `CharacterDelimitedCodec` with a maximum frame length limit.
+    /// Creates a `CharacterDelimitedCodec` with a maximum frame length limit.
+    ///
+    /// When more bytes than `max_length` have been read, all bytes will be
+    /// discarded until reaching the next delimiter.
```

Comment:
Similar to my below question 😄

Reply:
Changed in b2220aa94048b06450af996eb8222aa675947d07